### ISSUE TITLE
Update 'Copy & paste' to 'Copy & Paste' on Release Notes

### ISSUE THREAD

**Issue by v-xianya** [microsoft/AzureStorageExplorer#2279]
**Storage Explorer Version:** 1.11.1
**Build:** [20191119.3](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3255785&view=results)
**Branch:** hotfix/1.11.1
**Platform/OS:** Windows 10/ Linux Ubuntu 18.04/macOS High Sierra
**Architecture:** ia32/x64
**Regression From:** Not a regression

**Steps to reproduce:**
1. Launch Storage Explorer -> Help -> View Release Notes.
2. Check the first part of New section.

**Expect Experience:**
Show 'Copy & Paste'.

**Actual Experience:**
Show 'Copy & paste'.
![image](https://user-images.githubusercontent.com/41351993/69134310-412c3100-0af2-11ea-81c1-ffa01447a9ef.png)

**Comment by MRayermannMSFT** [comment]
We're happy with the release notes as is for now. But thanks for being vigilant. Won't fix.

**Comment by v-xianya** [comment]
Hi @MRayermannMSFT 
 I think it is better to use 'Copy & Paste' 
1. They are two operations in parallel.
2. Using 'Copy & Paste' in the below part.
![image](https://user-images.githubusercontent.com/41351993/69307271-efada000-0bde-11ea-820f-b63e68d37b95.png)
